orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -478,4 +480,11 @@ class OperationsController {
       pipeline.origin = AuthenticatedRequest.spinnakerUserOrigin.orElse('unknown')
     }
   }
+
+  @ExceptionHandler(UserException.class)
+  static void handleUserExceptions(UserException e, HttpServletResponse response) {
```

Comment:
do our generic error handlers in kork not handle UserException as a 400 already?

I think if they do, they will serve a json payload with some details instead of a text response - might play slightly nicer with handling API responses if they come back that way

Reply:
oh... i didn't see it let me look/try, thanks!

Reply:
doesn't look like it, but i can add one to kork, that will obviate the need for them here